server/hotrod/pom.xml

```diff
@@ -153,6 +153,10 @@
                               <arg value="${project.build.testOutputDirectory}/password_client_truststore.p12"/>
                               <arg value="-deststorepass"/>
                               <arg value="secret"/>
+                              <arg value="-srcstoretype"/>
+                              <arg value="pkcs12"/>
+                              <arg value="-deststoretype"/>
+                              <arg value="pkcs12"/>
```

Comment:
Do we not want "JCEKS" as the other one in client hotrod?

Reply:
@wburns I don't know. According to the last commits in server/hotrod all keystores were changed to pkcs12. I have just added the required part for IBM JDK to be able to build them.

Reply:
Okay, it seems to be fine with our CI, but I haven't confirmed with IBM, so I trust you guys on that front :)